### DISCUSSION TITLE
API setSubject

### DISCUSSION THREAD

**Discussion by AppDesv** [LiveHelperChat/livehelperchat#2058] [General]
Certainly! Below is the text translated into English:

"Is it possible through the API to update a chat topic?

I've tried something like this and other ways, but everything without success. Could you help me with this?"

API REQUEST: PUT[/restapi/chat/{chat_id}]

`{
 "type": "command",
        "content": {
            "text": "",
            "command": "setsubject",
            "payload": "38"
        }
}`

OR

`{
  "nick": "Test Upgrade Nick OK",
  "device_type": "0",
  "fields": {
    "setsubject": 38
  }
}`

**Comment by remdex** [comment]
Hi,
At the moment there is no API for subjects https://api.livehelperchat.com/#/ but you can make one. I would review and accept a pull request. Or you can always sponsor those things.
